core/modules/comment/comment.module

```diff
@@ -1167,6 +1168,18 @@ function comment_form_node_type_form_alter(&$form, $form_state) {
       '#title' => t('Allow comment title'),
       '#default_value' => $node_type->settings['comment_subject_field'],
     );
+    $form['comment']['comment_submitted_format'] = array(
+      '#type' => 'textfield',
+      '#title' => t('Comment author/date format'),
+      '#default_value' => $node_type->settings['comment_submitted_format'],
```

Comment:
This should be wrapped in `filter_xss_admin()` I think:

```suggestion
      '#default_value' => filter_xss_admin($node_type->settings['comment_submitted_format']),
```

Reply:
Why `_admin()` and not `filter_xss()`?

Reply:
Default values should not be modified, they should contain exactly what the user saved previously. So while we run `filter_xss_admin()` on output, it shouldn't be added here.